api.md

```diff
@@ -840,7 +842,127 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
 ---
+## Continuous Aggregates :community_function: [](continuous-aggregates)
+TimescaleDB allows users the ability to automatically recompute aggregates 
+at predefined intervals and materialize the results. This is suitable for 
+frequently used queries.
+
+## CREATE VIEW 
```

Comment:
I'd instead do `CREATE VIEW ... WITH (timescaledb.continuous)` here

Reply:
agreed with David

Reply:
@cevian @davidkohn88 It looks odd as a section header . How about CREATE VIEW for continuous aggregates and similarly rename the other sections the same way i.e. ALTER VIEW for continuous aggregates.

Reply:
leaving header as is and changed reference in the command list.